### PR TITLE
Add Logging quickstart sample

### PR DESCRIPTION
**Pull request by jmdobry**
@tswast or @jerjou Could one of you add a test for this?

### CODE REVIEW

File: datastore/cloud-client/src/main/java/com/example/datastore/QuickstartSample.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 public class QuickstartSample {
   public static void main(String... args) throws Exception {
     // Instantiates a client
-    Datastore datastore = DatastoreOptions.defaultInstance().service();
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
```
Comment (jmdobry) [inline datastore/cloud-client/src/main/java/com/example/datastore/QuickstartSample.java]: Should this commit be in this PR? I'm guessing yes?

**Comment by lesv** [comment]
@frankyn Can you add tests to this sample?   Also, it would be nice if you hooked it up to both (separate examples) java.util.logging & SLF4J.

**Comment by frankyn** [comment]
Les, is the following required for this PR? it would be nice if you hooked it up to both (separate examples) java.util.logging & SLF4J."

**Comment by frankyn** [comment]
Added Logging Quickstart to the **root** pom.xml

File: logging/cloud-client/src/test/java/com/example/logging/QuickstartSampleIT.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+    deleteMyLog();
+  }
+
+  @Test
+  public void testQuickstart() throws Exception {
+    QuickstartSample.main("my-log");
```
Comment (lesv) [inline logging/cloud-client/src/test/java/com/example/logging/QuickstartSampleIT.java]: This may need to wait for data -- it might not be there immediatly

Comment (tswast) [inline logging/cloud-client/src/test/java/com/example/logging/QuickstartSampleIT.java]: The test is reading stdout, not Cloud Logging. While it'd be nice if the quickstart also showed how to read logs, I think that's beyond the scope of a quickstart. @jmdobry thoughts?

Comment (lesv) [inline logging/cloud-client/src/test/java/com/example/logging/QuickstartSampleIT.java]: Apparently I wasn't paying enough attention.

**Comment by lesv** [comment]
confirm

**Comment by jmdobry** [comment]
Sorry, not ready to delete branch.

**Pull request merged**